private/buf/buffetch/internal/reader.go

```diff
@@ -361,6 +363,9 @@ func (r *reader) getProtoFileBucket(
 	if !r.localEnabled {
 		return nil, NewReadLocalDisabledError()
 	}
+	if datawkt.Exists(protoFileRef.Path()) {
+		return wellKnownTypeBucket(ctx)
```

Comment:
Why would I return the entire bucket if I have a single file? Shouldn't `buf beta convert google/protobuf/timestamp.proto --type google.type.Duration` fail?

Reply:
The compiler's source accessor has the logic for falling back when needed. So maybe the change needed is in the file provider for OS, below, to simply _allow_ "file not found" issues for paths that exist in `datawkt`. WDYT?